app/services/notify_app_store.rb

```diff
@@ -0,0 +1,40 @@
+class NotifyAppStore
+  attr_reader :notifier, :message_builder
+
+  def initialize(claim:, scorer: NotifyAppStore::Scorer)
+    @message_builder = MessageBuilder.new(claim:, scorer:)
+    @notifier = notifier
+  end
+
+  def process
+    raise 'Sidekiq workers is not yet enabled' if ENV.key?('REDIS_URL')
```

Comment:
```suggestion
    raise 'Sidekiq workers is not yet enabled' if ENV.key?('REDIS_HOST')
```

Reply:
so the ruby backed redis libraries will automatically config when a `REDIS_URL` is present... I don't think that works with `REIS_HOST`...

is there a specific reason for this request?

Reply:
As discussed, previous projects we have developed used the REDIS_URL which came from infra code however we came unstuck when a major version of Redis changed how the url was formed (I think it was a username issue) so we moved the formation into the application to give us more power in the future.